ui/src/api/index.ts

```diff
@@ -0,0 +1,8 @@
+import AuthApi from './auth-api';
+import HealthCheckApi from './healthcheck-api';
+
+export default class ApiService {
+  static getAuthApi = () => AuthApi;
```

Comment:
I'd probably use a single static method (e.g. getService) that accepts a key and use a Map of services that can be accessed by a passed key. So we won't have to modify the ApiService at all.